src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt

```diff
@@ -102,6 +102,9 @@ data class ChannelClosing(val channelId: ByteVector32) : PeerEvent()
 data class SendSwapOutRequest(val amount: Satoshi, val bitcoinAddress: String, val feePerKw: Long) : PeerCommand()
 data class SwapOutResponseEvent(val swapOutResponse: SwapOutResponse) : PeerEvent()
 
+data class SendPhoenixAndroidLegacyMigrate(val newNodeId: PublicKey) : PeerCommand()
```

Comment:
`SendPhoenixAndroidLegacyMigrate` is a message sent by the phoenix-legacy app to let the peer know when the user started the migration process, so that the peer could start the required migration logic on its side, if there's any.

With dual funding I think this message is not needed anymore as the process is stateless on the peer side.

Reply:
Done with [f202d93](https://github.com/ACINQ/lightning-kmp/pull/383/commits/f202d93dfdaa0b427cd04ecc1cfc5de437650d74).